rel-eng/tito.props

```diff
@@ -1105,3 +1105,13 @@ whitelist = jsoncpp
   rubygem-typhoeus
   rubygem-zest
   saslwrapper
+
+
+[foreman-plugin-nightly-rhel7]
+whitelist = 
+        rubygem-hammer_cli_foreman_puppet
+
+
+[foreman-plugin-nightly-el8]
+whitelist = 
+        rubygem-hammer_cli_foreman_puppet
```

Comment:
This isn't right

Reply:
the `add_gem` script is broken, https://github.com/theforeman/foreman-packaging/pull/6972 should fix it